esp/useful_scripts/server_setup/new_site.sh

```diff
@@ -269,13 +271,14 @@ CACHE_PREFIX = "${SITENAME}ESP"
 DEFAULT_EMAIL_ADDRESSES = {
         'archive': 'learninguarchive@gmail.com',
         'bounces': 'learningubounces@gmail.com',
-        'support': '$SITENAME-websupport@learningu.org',
+        'support': '$GROUPEMAIL',
         'membership': '$GROUPEMAIL',
         'default': '$GROUPEMAIL',
         }
 ORGANIZATION_SHORT_NAME = '$GROUPNAME'
 INSTITUTION_NAME = '$INSTITUTION'
 EMAIL_HOST = '$EMAILHOST'
+EMAIL_HOST_SENDER = EMAIL_HOST
```

Comment:
Oh, I see, I got confused because this is actually python, but with bash variable interpolation.  You're right.